frontend/ui/elements/reader_menu_order.lua

```diff
@@ -124,6 +124,7 @@ local order = {
         "news_downloader",
         "send2ebook",
         "text_editor",
+        "profiles",
```

Comment:
How does this affect the paging?

Reply:
![Reader_2020-06-17_170154](https://user-images.githubusercontent.com/38916402/84950126-5daa3f00-b0bc-11ea-8c47-16ddf19eda6d.png)

Reply:
That probably means it pushes plugin management to the next page on 3:2 devices. But wouldn't it be better in something like taps & gestures regardless?

Reply:
I put it there by default (this type of thing was discussed in another issue I believe). I am happy to put it somewhere else, once everyone voices their opinion. I only want to move it once :)

Reply:
For onlookers, that would be https://github.com/koreader/koreader/issues/6105#issuecomment-621653800